Projects/Roman2Dec-App.md

```diff
@@ -0,0 +1,51 @@
+# Roman2Dec
+
+**Tier:** 1-Intermediate
+
+The purpose of this app is to allow the user put in practice both, algorithmic and Web Development knowledge by creating a Roman to Decimal numbers converter app.
+
+The numeric system represented by Roman numerals originated in ancient Rome and remained the 
+usual way of writing numbers throughout Europe well into the Late Middle Ages. 
+Roman numerals, as used today, employ seven symbols, each with a fixed integer value.
+
+See the below table with *Symbol - Value*
+ 	
+ 	I - 1
+ 	V - 5
+ 	X - 10
+ 	L - 50
+ 	C - 100
+ 	D - 500
+ 	M - 1000
+
+Please take a look in Wikipedia [for more details about Roman Numbers](https://en.wikipedia.org/wiki/Roman_numerals)
```

Comment:
I would remove this link too as it is in the Useful Links section down below!

Reply:
Done!